### PR TITLE
docs(dropdown): add demo for divider and separated link and change name of DemoDropupComponent

### PR DESCRIPTION
**Pull request by ghost**
-add demo for divider and separated link
joxi_screenshot_1519036401041

-change name of DemoDropupComponent (like other components)
 
Related to #3788

### CODE REVIEW

File: demo/src/app/components/+dropdown/demos/dropup/dropup.ts [hunk 1]
```diff
@@ -1,9 +0,0 @@
-import { Component } from '@angular/core';
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/demos/dropup/dropup.ts]: why did you remove dropup demo?

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+dropdown/demos/dropup/dropup.ts (hunk 1 above)
Comment (ghost) [reply demo/src/app/components/+dropdown/demos/dropup/dropup.ts]: cause I've added it in another pr (issue № #3790)

File: demo/src/app/components/+dropdown/demos/dropup/dropup.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { Component } from '@angular/core';
 
 @Component({
-  selector: 'demo-dropup',
+  selector: 'demo-dropdown-dropup',
   templateUrl: './dropup.html'
 })
 export class DemoDropupComponent {
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/demos/dropup/dropup.ts]: change the name of class as well if you changed selector

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Please, rename toggler button, because this dropdown is not dropup (and couldn't appear like it)
needrename
Otherwise, looks good

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — CHANGES_REQUESTED** [review]
1. special demo for divider - ok
2. removing dividers from all other places doesn't make sense

**Comment by ghost** [comment]
return divider and separated link in other component

**Review by valorkin — APPROVED** [review]
